ci/script.sh

```diff
@@ -17,12 +17,18 @@ if [ "$TRAVIS_RUST_VERSION" = "1.12.0" ]; then
   exit
 fi
 
-# Run tests. If we have nightly, then enable our nightly features.
+# Run tests.
+#
+# If we have nightly, then enable our nightly features.
+# If we are part of a nightly build, then run expensive random tests.
+CARGO_TEST_ARGS=""
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+    CARGO_TEST_ARGS="${CARGO_TEST_ARGS} --features random-test"
+fi
 if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
-  cargo test --verbose --features unstable
-else
-  cargo test --verbose
+    CARGO_TEST_ARGS="${CARGO_TEST_ARGS} --features unstable"
 fi
+cargo test --verbose ${CARGO_TEST_ARGS}
```

Comment:
Why did the else block get removed here? I guess it's probably OK, but this will cause the nightly CI test to get quite a bit longer.

Reply:
Well my plan was to just build up an arg string in the if statements so that random testing was handled seperatly from the unstable feature checking, then execute a single cargo test will all the args, but now I realize that this would cause two `--features` flags to be passed :(. This is going away as part of replacing the feature with an env var anyway.